### PR TITLE
General updates

### PR DESCRIPTION
**Pull request by bjd2385**

### CODE REVIEW

File: src/scripts/continue.sh [hunk 1]
```diff
@@ -1,24 +1,22 @@
-#!/bin/sh
-
-set -e
```
Comment (bjd2385) [inline src/scripts/continue.sh:3]: Noticed this appearing in my jobs, no need for a shebang CCI adds them automatically.

```shell
#!/bin/bash -eo pipefail
#!/bin/sh

set -e
...
```

+ it's already set.

Comment (bjd2385) [reply src/scripts/continue.sh:3]: https://app.circleci.com/pipelines/github/bjd2385/dynamic-continuation-orb/302/workflows/a9000cb6-64ae-4124-8064-29cf26f6e189/jobs/812?invite=true#step-108-1

File: src/scripts/continue.sh [hunk 2]
```diff
@@ -31,7 +29,7 @@ if ! $COMMAND; then
 fi
 
 mkdir -p /tmp/circleci
-rm -rf /tmp/circleci/continue_post.json
+rm /tmp/circleci/continue_post.json
```
Comment (stig) [inline src/scripts/continue.sh:32]: Good catch!

File: src/scripts/continue.sh [hunk 3]
```diff
@@ -52,4 +50,4 @@ cat /tmp/circleci/continue_post.json
         -H "Accept: application/json"  \
         --data @/tmp/circleci/continue_post.json \
         "https://${CIRCLECI_DOMAIN}/api/v2/pipeline/continue")" \
-   -eq 200 ]
+   -eq 200 ]
```
Comment (stig) [inline src/scripts/continue.sh:53]: Add back final newline please.

File: src/scripts/finish.sh [hunk 4]
```diff
@@ -1,25 +1,27 @@
...
+[ $(curl \
         -o /dev/stderr \
         -w '%{http_code}' \
         -XPOST \
         -H "Content-Type: application/json" \
         -H "Accept: application/json"  \
         --data "${JSON_BODY}" \
         "https://${CIRCLECI_DOMAIN}/api/v2/pipeline/continue") \
-   -eq 200 ]]
+   -eq 200 ]
```
Comment (stig) [inline src/scripts/finish.sh:27]: Please can you add back the final newline?

File: .pre-commit-config.yaml
Comment (stig) [inline .pre-commit-config.yaml:1]: Could you remove this file from this PR, but submit it separately? I think it's interesting, but I've never encountered these pre-packaged pre-commit-hook repos and need time to consider whether that's appropriate for us. I don't want discussions around that to distract from the other good stuff in this PR.

File: .pre-commit-config.yaml [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+      - id: trailing-whitespace
+      - id: check-added-large-files
+        args: [--maxkb=10000, --enforce-all]
+      - id: check-executables-have-shebangs
+      - id: check-shebang-scripts-are-executable
+      - id: mixed-line-ending
+      - id: trailing-whitespace
```
Comment (stig) [inline .pre-commit-config.yaml:12]: This line is a duplicate of line 6. If you do a new PR for this, as I requested elsewhere, I would recommend replacing it with `end-of-file-fixer`.

**Review by stig — COMMENTED** [review]
Thank you for contributing! I've left some comments for you to consider.

FWIW I think there's a bit of overlap here with https://github.com/CircleCI-Public/continuation-orb/pull/35 -- which we are likely to merge within a week. Please rebase this once that lands.

**Comment by bjd2385** [comment]
Wait, that was a mistake

**Comment by bjd2385** [comment]
@stig can you re-open this pls :pray:

**Comment by Markl121** [comment]
Oh no, it looks like there are no commits from the branch `bjd2385:master`. I think we can only reopen the PR once the changes are brought back

**Pull request closed without merging**